scripts/wait-for.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# poll the proxy endpoint until we get a 0 return code or 5mins have passed, in that case exit 1
+timeout_time=$(($(date +%s) + 300))
```

Comment:
prob 2 mins is enough? I personally prefer cis to crash fast and I can restart them manually (restarting on a separate machine might also fix a problem if it was github/azure related)